osrm.nix

```diff
@@ -1,4 +1,4 @@
-{ inputs, ... }:
+{ osrmBackend, inputs, ... }:
```

Comment:
Why do you need this, when it is literally in the `inputs` argument (as `inputs.osrm-backend`)?